lib/ssh/scp.go

```diff
@@ -0,0 +1,88 @@
+package sshutils
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/gravitational/robotest/lib/utils"
+
+	"github.com/gravitational/trace"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// CopyFile transfers local file to remote host directory
+func PutFile(ctx context.Context, logFn utils.LogFnType, client *ssh.Client, srcPath, dstDir string) (remotePath string, err error) {
+	session, err := client.NewSession()
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+	defer session.Close()
+
+	f, err := os.Open(srcPath)
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+	defer f.Close()
+
+	fi, err := os.Stat(srcPath)
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+
+	errCh := make(chan error, 2)
+	go scpSendFile(session, f, fi, errCh)
+
+	go func() {
+		err := session.Run(fmt.Sprintf("/usr/bin/scp -tr %s", dstDir))
+		errCh <- err
```

Comment:
this error will not have any trace. I guess it will be harder to find the problem